infrastructure/stacks/terraform_management/provider.tf

```diff
@@ -17,9 +17,9 @@ provider "aws" {
 
   default_tags {
     tags = {
-      owner               = "UEC"
-      project             = "UEC DoS"
-      environment         = "Dev"
+      owner               = var.project_owner
+      project             = var.project
```

Comment:
this will be set to nhse-uec-dos
Do we want to be able to differentiate between dos and cm?

Reply:
Yes interesting... the common.tfvars file may not be sufficient - we may need to have a common dos and a common cm or something else... will ponder

Reply:
I have created a dos and a cm project.tfvars